client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -401,7 +410,11 @@ const InvestigationTable: React.FC = (): JSX.Element => {
     }, [tableRows, unassignedInvestigationsCount]);
 
     return (
-        <>
+        <div tabIndex={0}
+            onClick={() => closeDropdowns()}
```

Comment:
i think you can do it like this: 
onClick={closeDropDowns}, and not defining a function